### DISCUSSION TITLE
Playground example codes

### DISCUSSION THREAD

**Discussion by andersontz** [shadcn-ui/ui#3538] [General]
One thing I didn't understand is how I get the samples codes (Mail, Dashboard, Cards, and Others), which already have something ready, to start my development on. It's possible? How do I download? They all have some interesting part for my development, such as login screen, slideable sidebar, dashboard, etc.

**Comment by fry69** [comment]
Found the link in another discussion: https://github.com/shadcn-ui/ui/tree/main/apps/www/app/(app)/examples

This seems like it is an unanswered FAQ, it should be mentioned somewhere how to get/see the code for the examples, IMHO.

**Comment by thalesfsp** [comment]
Still, ,how to run the examples? For example, the email one?
